util/tracing/transform/instrumentation.go

```diff
@@ -6,12 +6,12 @@ import (
 	"go.opentelemetry.io/otel/sdk/instrumentation"
 )
 
-func instrumentationLibrary(il *commonpb.InstrumentationLibrary) instrumentation.Library {
```

Comment:
Looks like this library has been updated a lot in the upstream. Please follow-up with full update so we get all the changes.

Reply:
- https://github.com/containerd/containerd/pull/8026

Reply:
?

You meand you want to update otel again before addressing this?

Atm buildkit has copy of otel library https://github.com/moby/buildkit/tree/master/util/tracing/transform . We can't link upstream because that package was internal. Unless there is now a better way to do this we should keep this package up to date when we update otel in vendor.

Reply:
Sorry, didn't get what you meant by "full update", but looks like you meant copying https://github.com/open-telemetry/opentelemetry-go/tree/main/exporters/otlp/otlptrace/internal/tracetransform to https://github.com/moby/buildkit/tree/master/util/tracing/transform